src/complex/Filter/Output.cpp

```diff
@@ -132,4 +133,26 @@ DataPath CreateArrayAction::path() const
 {
   return m_Path;
 }
+
+//------------------------------------------------------------------------------
+CreateDataGroupAction::CreateDataGroupAction(const DataPath& path)
+: m_Path(path)
+{
+}
+
+CreateDataGroupAction::~CreateDataGroupAction() noexcept = default;
+
+DataPath CreateDataGroupAction::parentPath() const
+{
+  return m_Path;
+}
+
+Result<> CreateDataGroupAction::apply(DataStructure& dataStructure, Mode mode) const
+{
+  auto linkedPath = dataStructure.makePath(m_Path);
+
+  // Assuming nothing went wrong, return an empty Result<> object
+  return {};
```

Comment:
We should report errors here.

Reply:
makePath is going to need to report errors.